
From: Christoph Hellwig <hch@lst.de>

Rather lowlevel functions that modules shouldn't mess with and foruntatly
currently don't.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/file_table.c |    6 +-----
 1 files changed, 1 insertion(+), 5 deletions(-)

diff -puN fs/file_table.c~unexport-files_lock-and-put_filp fs/file_table.c
--- 25/fs/file_table.c~unexport-files_lock-and-put_filp	Tue Sep  7 16:29:36 2004
+++ 25-akpm/fs/file_table.c	Tue Sep  7 16:29:36 2004
@@ -24,11 +24,9 @@ struct files_stat_struct files_stat = {
 
 EXPORT_SYMBOL(files_stat); /* Needed by unix.o */
 
-/* public *and* exported. Not pretty! */
+/* public. Not pretty! */
 spinlock_t __cacheline_aligned_in_smp files_lock = SPIN_LOCK_UNLOCKED;
 
-EXPORT_SYMBOL(files_lock);
-
 static spinlock_t filp_count_lock = SPIN_LOCK_UNLOCKED;
 
 /* slab constructors and destructors are called from arbitrary
@@ -199,8 +197,6 @@ void put_filp(struct file *file)
 	}
 }
 
-EXPORT_SYMBOL(put_filp);
-
 void file_move(struct file *file, struct list_head *list)
 {
 	if (!list)
_
